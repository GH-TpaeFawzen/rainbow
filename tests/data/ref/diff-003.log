--- setup.cfg	2017-03-18 22:27:04.430726575 +0100
+++ setup.py	2017-03-19 11:54:17.386754769 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 # ----------------------------------------------------------------------
 # rainbow, a terminal colorizer - https://github.com/nicoulaj/rainbow
 # copyright (c) 2010-2018 rainbow contributors
@@ -16,24 +17,60 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 # ----------------------------------------------------------------------
 
-[clean]
-paths = .cache,
-        .tox,
-        build,
-        dist,
-        rainbow.egg-info,
-        .coverage
+import os
 
-[build_completion_bash]
-shell = bash
-output = build/completion/rainbow
+from setuptools import setup, find_packages
 
-[build_completion_zsh]
-shell = zsh
-output = build/completion/_rainbow
+from rainbow import VERSION
+from rainbow.build import Clean, Build, GenerateCompletion, GenerateManPage
 
-[build_man_page]
-output = build/man/rainbow.1
-
-[bdist_wheel]
-universal = 1
+setup(
+    name='rainbow',
+    version=VERSION,
+    author='Julien Nicoulaud',
+    author_email='julien.nicoulaud@gmail.com',
+    description='Colorize commands output or STDIN using patterns.',
+    long_description=open(os.path.join(os.path.dirname(__file__), 'README.rst')).read(),
+    license='GPLv3',
+    url='https://github.com/nicoulaj/rainbow',
+    keywords='color colorize colorizer pattern',
+    classifiers=[
+        'Development Status :: 5 - Production/Stable',
+        'Environment :: Console',
+        'Operating System :: MacOS :: MacOS X',
+        'Operating System :: POSIX',
+        'Topic :: System',
+        'Topic :: Utilities',
+        'Intended Audience :: Developers',
+        'Intended Audience :: End Users/Desktop',
+        'Intended Audience :: System Administrators',
+        'Natural Language :: English',
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.6',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.3',
+        'Programming Language :: Python :: 3.4',
+        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: Implementation :: CPython',
+        'Programming Language :: Python :: Implementation :: PyPy',
+        'License :: OSI Approved :: GNU General Public License (GPL)'
+    ],
+    packages=find_packages(exclude=['tests*']),
+    include_package_data=True,
+    data_files=[
+        ('/etc/bash_completion.d', ['build/completion/rainbow']),
+        ('/usr/share/zsh/site-functions', ['build/completion/_rainbow']),
+        ('/usr/share/man/man1', ['build/man/rainbow.1.gz'])
+    ],
+    scripts=['scripts/rainbow'],
+    cmdclass={
+        'clean': Clean,
+        'build': Build,
+        'build_completion_bash': GenerateCompletion,
+        'build_completion_zsh': GenerateCompletion,
+        'build_man_page': GenerateManPage
+    }
+)
