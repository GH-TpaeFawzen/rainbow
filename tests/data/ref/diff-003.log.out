[FOREGROUND_RED]--- setup.cfg	2017-03-18 22:27:04.430726575 +0100[FOREGROUND_RESET]
[FOREGROUND_GREEN]+++ setup.py	2017-03-19 11:54:17.386754769 +0100[FOREGROUND_RESET]
[FOREGROUND_MAGENTA]@@ -1,3 +1,4 @@[FOREGROUND_RESET]
[FOREGROUND_GREEN]+#!/usr/bin/env python[FOREGROUND_RESET]
[FAINT] # ----------------------------------------------------------------------[RESET_INTENSITY]
[FAINT] # rainbow, a terminal colorizer - https://github.com/nicoulaj/rainbow[RESET_INTENSITY]
[FAINT] # copyright (c) 2010-2017 rainbow contributors[RESET_INTENSITY]
[FOREGROUND_MAGENTA]@@ -16,24 +17,60 @@[FOREGROUND_RESET]
[FAINT] # along with this program.  If not, see <http://www.gnu.org/licenses/>.[RESET_INTENSITY]
[FAINT] # ----------------------------------------------------------------------[RESET_INTENSITY]
[FAINT] [RESET_INTENSITY]
[FOREGROUND_RED]-[clean][FOREGROUND_RESET]
[FOREGROUND_RED]-paths = .cache,[FOREGROUND_RESET]
[FOREGROUND_RED]-        .tox,[FOREGROUND_RESET]
[FOREGROUND_RED]-        build,[FOREGROUND_RESET]
[FOREGROUND_RED]-        dist,[FOREGROUND_RESET]
[FOREGROUND_RED]-        rainbow.egg-info,[FOREGROUND_RESET]
[FOREGROUND_RED]-        .coverage[FOREGROUND_RESET]
[FOREGROUND_GREEN]+import os[FOREGROUND_RESET]
[FAINT] [RESET_INTENSITY]
[FOREGROUND_RED]-[build_completion_bash][FOREGROUND_RESET]
[FOREGROUND_RED]-shell = bash[FOREGROUND_RESET]
[FOREGROUND_RED]-output = build/completion/rainbow[FOREGROUND_RESET]
[FOREGROUND_GREEN]+from setuptools import setup, find_packages[FOREGROUND_RESET]
[FAINT] [RESET_INTENSITY]
[FOREGROUND_RED]-[build_completion_zsh][FOREGROUND_RESET]
[FOREGROUND_RED]-shell = zsh[FOREGROUND_RESET]
[FOREGROUND_RED]-output = build/completion/_rainbow[FOREGROUND_RESET]
[FOREGROUND_GREEN]+from rainbow import VERSION[FOREGROUND_RESET]
[FOREGROUND_GREEN]+from rainbow.build import Clean, Build, GenerateCompletion, GenerateManPage[FOREGROUND_RESET]
[FAINT] [RESET_INTENSITY]
[FOREGROUND_RED]-[build_man_page][FOREGROUND_RESET]
[FOREGROUND_RED]-output = build/man/rainbow.1[FOREGROUND_RESET]
[FOREGROUND_RED]-[FOREGROUND_RESET]
[FOREGROUND_RED]-[bdist_wheel][FOREGROUND_RESET]
[FOREGROUND_RED]-universal = 1[FOREGROUND_RESET]
[FOREGROUND_GREEN]+setup([FOREGROUND_RESET]
[FOREGROUND_GREEN]+    name='rainbow',[FOREGROUND_RESET]
[FOREGROUND_GREEN]+    version=VERSION,[FOREGROUND_RESET]
[FOREGROUND_GREEN]+    author='Julien Nicoulaud',[FOREGROUND_RESET]
[FOREGROUND_GREEN]+    author_email='julien.nicoulaud@gmail.com',[FOREGROUND_RESET]
[FOREGROUND_GREEN]+    description='Colorize commands output or STDIN using patterns.',[FOREGROUND_RESET]
[FOREGROUND_GREEN]+    long_description=open(os.path.join(os.path.dirname(__file__), 'README.rst')).read(),[FOREGROUND_RESET]
[FOREGROUND_GREEN]+    license='GPLv3',[FOREGROUND_RESET]
[FOREGROUND_GREEN]+    url='https://github.com/nicoulaj/rainbow',[FOREGROUND_RESET]
[FOREGROUND_GREEN]+    keywords='color colorize colorizer pattern',[FOREGROUND_RESET]
[FOREGROUND_GREEN]+    classifiers=[[FOREGROUND_RESET]
[FOREGROUND_GREEN]+        'Development Status :: 5 - Production/Stable',[FOREGROUND_RESET]
[FOREGROUND_GREEN]+        'Environment :: Console',[FOREGROUND_RESET]
[FOREGROUND_GREEN]+        'Operating System :: MacOS :: MacOS X',[FOREGROUND_RESET]
[FOREGROUND_GREEN]+        'Operating System :: POSIX',[FOREGROUND_RESET]
[FOREGROUND_GREEN]+        'Topic :: System',[FOREGROUND_RESET]
[FOREGROUND_GREEN]+        'Topic :: Utilities',[FOREGROUND_RESET]
[FOREGROUND_GREEN]+        'Intended Audience :: Developers',[FOREGROUND_RESET]
[FOREGROUND_GREEN]+        'Intended Audience :: End Users/Desktop',[FOREGROUND_RESET]
[FOREGROUND_GREEN]+        'Intended Audience :: System Administrators',[FOREGROUND_RESET]
[FOREGROUND_GREEN]+        'Natural Language :: English',[FOREGROUND_RESET]
[FOREGROUND_GREEN]+        'Programming Language :: Python',[FOREGROUND_RESET]
[FOREGROUND_GREEN]+        'Programming Language :: Python :: 2',[FOREGROUND_RESET]
[FOREGROUND_GREEN]+        'Programming Language :: Python :: 2.6',[FOREGROUND_RESET]
[FOREGROUND_GREEN]+        'Programming Language :: Python :: 2.7',[FOREGROUND_RESET]
[FOREGROUND_GREEN]+        'Programming Language :: Python :: 3',[FOREGROUND_RESET]
[FOREGROUND_GREEN]+        'Programming Language :: Python :: 3.3',[FOREGROUND_RESET]
[FOREGROUND_GREEN]+        'Programming Language :: Python :: 3.4',[FOREGROUND_RESET]
[FOREGROUND_GREEN]+        'Programming Language :: Python :: 3.5',[FOREGROUND_RESET]
[FOREGROUND_GREEN]+        'Programming Language :: Python :: 3.6',[FOREGROUND_RESET]
[FOREGROUND_GREEN]+        'Programming Language :: Python :: Implementation :: CPython',[FOREGROUND_RESET]
[FOREGROUND_GREEN]+        'Programming Language :: Python :: Implementation :: PyPy',[FOREGROUND_RESET]
[FOREGROUND_GREEN]+        'License :: OSI Approved :: GNU General Public License (GPL)'[FOREGROUND_RESET]
[FOREGROUND_GREEN]+    ],[FOREGROUND_RESET]
[FOREGROUND_GREEN]+    packages=find_packages(exclude=['tests*']),[FOREGROUND_RESET]
[FOREGROUND_GREEN]+    include_package_data=True,[FOREGROUND_RESET]
[FOREGROUND_GREEN]+    data_files=[[FOREGROUND_RESET]
[FOREGROUND_GREEN]+        ('/etc/bash_completion.d', ['build/completion/rainbow']),[FOREGROUND_RESET]
[FOREGROUND_GREEN]+        ('/usr/share/zsh/site-functions', ['build/completion/_rainbow']),[FOREGROUND_RESET]
[FOREGROUND_GREEN]+        ('/usr/share/man/man1', ['build/man/rainbow.1.gz'])[FOREGROUND_RESET]
[FOREGROUND_GREEN]+    ],[FOREGROUND_RESET]
[FOREGROUND_GREEN]+    scripts=['scripts/rainbow'],[FOREGROUND_RESET]
[FOREGROUND_GREEN]+    cmdclass={[FOREGROUND_RESET]
[FOREGROUND_GREEN]+        'clean': Clean,[FOREGROUND_RESET]
[FOREGROUND_GREEN]+        'build': Build,[FOREGROUND_RESET]
[FOREGROUND_GREEN]+        'build_completion_bash': GenerateCompletion,[FOREGROUND_RESET]
[FOREGROUND_GREEN]+        'build_completion_zsh': GenerateCompletion,[FOREGROUND_RESET]
[FOREGROUND_GREEN]+        'build_man_page': GenerateManPage[FOREGROUND_RESET]
[FOREGROUND_GREEN]+    }[FOREGROUND_RESET]
[FOREGROUND_GREEN]+)[FOREGROUND_RESET]
